### PR TITLE
Add missing align_image_stack to the distribution

### PR DESCRIPTION
**Pull request by anxolerd**
Issue ref: https://github.com/LuminanceHDR/LuminanceHDR/issues/140

This one is the attempt to add `align_image_stack` binary to the flathub distribution of LHDR.
While this should work, it needs some polishing and cleaning up and (if possible) optimizations.

### CODE REVIEW

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 1]
```diff
@@ -109,6 +109,80 @@ modules:
...
+  - name: vigra
+    buildsystem: cmake-ninja
+    config-opts:
+       - -DWITH_OPENEXR=ON
+    sources:
+      - type: archive
+        url: https://github.com/ukoethe/vigra/archive/Version-1-9-0.tar.gz
+        sha256: dc041f7ccf838d4321e9bcf522fece1758768dd7a3f8350d1e83e2b8e6daf1e6
+
+  - name: glew
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Try to reuse https://github.com/flathub/shared-modules for glew

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 2]
```diff
@@ -109,6 +109,80 @@ modules:
...
+      - "GLEW_PREFIX=/app"
+      - "GLEW_DEST=/app"
+      - "LIBDIR=/app/lib"
+    make-install-args: *glew-make-args
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/glew/glew/2.1.0/glew-2.1.0.tgz
+        sha256: 04de91e7e6763039bc11940095cd9c7f880baba82196a7765f727ac05a993c95
+
+  - name: glu
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Try to reuse https://github.com/flathub/shared-modules for glu

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 3]
```diff
@@ -109,6 +109,80 @@ modules:
...
+      - --with-libiconv=/usr
+    build-options:
+      cxxflags: "-std=c++0x"
+    sources:
+      - type: archive
+        url: https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2
+        sha256: 96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+
+  - name: libpano13
+    buildsystem: cmake
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Maybe find a way to workaround cyclic dependency and use cmake-ninja

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 4]
```diff
@@ -109,6 +109,80 @@ modules:
...
+        sha256: 037357383978341dea8f572a5d2a0876c5ab0a83dffda431bd393357e91d95a8
+
+  - name: align_image_stack
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/hugin/hugin/hugin-2018.0/hugin-2018.0.0.tar.bz2
+        sha256: d3af0d066ac50e3bb243a175a64ecda136d87178419457e8822e11bcf0e565cb
+      #- type: patch
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: delete commented code

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 5]
```diff
@@ -109,6 +109,80 @@ modules:
...
+        sha256: 96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0
+
+  - name: libpano13
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/panotools/libpano13/libpano13-2.9.19/libpano13-2.9.19.tar.gz
+        sha256: 037357383978341dea8f572a5d2a0876c5ab0a83dffda431bd393357e91d95a8
+
+  - name: align_image_stack
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Find a way to compile only necessary, not the whole Hugin app

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml [hunk 6]
```diff
@@ -109,6 +109,80 @@ modules:
...
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/hugin/hugin/hugin-2018.0/hugin-2018.0.0.tar.bz2
+        sha256: d3af0d066ac50e3bb243a175a64ecda136d87178419457e8822e11bcf0e565cb
+      #- type: patch
+        #path: patches/hugin.patch
```
Comment (anxolerd) [inline net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Add cleanup to remove unused files.

Comment (fcomida) [reply net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: Is it ready to be merged?

Comment (anxolerd) [reply net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: It's not ready yet. :(

File: net.sourceforge.qtpfsgui.LuminanceHDR.yaml (hunk 5 above)
Comment (anxolerd) [reply net.sourceforge.qtpfsgui.LuminanceHDR.yaml]: From what I found so far it is impossible =(

**Comment by anxolerd** [comment]
@fcomida please take a look and merge if OK

**Comment by fcomida** [comment]
@anxolerd let's ask for a test bot, build

**Pull request merged**